concourse/scripts/self-update-pipeline.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Required variables are:
+# - DEPLOY_ENV
+# - AWS_ACCOUNT
+# - SELF_UPDATE_PIPELINE
+#
+# Optional variables:
+# - BRANCH
+
+set -u
+set -e
+
+if [ ! -d "./paas-bootstrap" ]; then
+  echo "Resource paas-bootstrap must be checked out"
+  exit 1
+fi
```

Comment:
This check seems redundant given this script is located within the paas-bootstrap repo.